package.json

```diff
@@ -24,7 +24,7 @@
     "tap": "^1.3.2"
   },
   "dependencies": {
-    "nan": "^2.0.5"
+    "nan": "^2.3.5"
```

Comment:
yes, what you probably noticed is that npm install does *not* update dependencies. so if you had previously installed, had an old nan, then installed again, nan would not be updated. then, I bet you did something like `npm install --save nan` or something of the like, and that caused it to update nan, and also update the package.json. the latter was irrelevant, the nan update was what helped you.

Anyhow, do this: `rm -rf node_modules; npm i` and you will find that you get latest versions.